components/hab/src/cli.rs

```diff
@@ -546,6 +547,15 @@ pub fn get() -> App<'static, 'static> {
                 )
             )
         )
+        (@subcommand sup =>
+            (@setting VersionlessSubcommands)
+            (@setting SubcommandRequiredElseHelp)
+            // these are the only sup cmds handled in `hab`, the remaining
+            // supervisor related commands are handled in `hab-sup`
+            (subcommand: sub_sup_depart().aliases(&["d", "de", "dep", "depa", "depart"]))
```

Comment:
This command is supposed to be undocumented, but with these changes it ends up displayed on `hab sup` when it wasn't before.

```
➤ env RUST_LOG= HAB_SUP_BINARY=(pwd)/target/debug/hab-sup target/debug/hab sup
hab-sup 0.60.0-dev

Authors: The Habitat Maintainers <humans@habitat.sh>
The Habitat Supervisor

USAGE:
    hab sup <SUBCOMMAND>

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information

SUBCOMMANDS:
    bash      Start an interactive Bash-like shell
    depart    Depart a Supervisor from the gossip ring; kicking and banning the target from joining again with the
              same member-id
    help      Prints this message or the help of the given subcommand(s)
    run       Run the Habitat Supervisor
    secret    Commands relating to a Habitat Supervisor's Contorl Gateway secret
    sh        Start an interactive Bourne-like shell
    status    Query the status of Habitat services.
    term      Gracefully terminate the Habitat Supervisor and all of its running services
```
vs
```
➤ env RUST_LOG= hab sup
hab-sup 0.59.0/20180712161546

Authors: The Habitat Maintainers <humans@habitat.sh>
The Habitat Supervisor

USAGE:
    hab-sup <SUBCOMMAND>

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information

SUBCOMMANDS:
    bash    Start an interactive Bash-like shell
    help    Prints this message or the help of the given subcommand(s)
    run     Run the Habitat Supervisor
    sh      Start an interactive Bourne-like shell
    term    Gracefully terminate the Habitat Supervisor and all of its running services
```

Reply:
This actually _should_ be documented... it's a frequently-asked question. The fact that it wasn't before was, I think, an unintended side-effect of how our CLI is currently split up.